access/src/main/external/static/templates/admin/editLabelForm.html

```diff
@@ -0,0 +1,18 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="edit" method="post" action="editlabel/<%= pid %>" enctype="multipart/form-data" target="upload_file_container" class="edit_form edit_label_form">
```

Comment:
The id attribute should be more descriptive, such as "edit_label_form".

Please also add a top margin to the form, as with the "Create Container" form: https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/edit-label/access/src/main/external/static/css/admin/admin_forms.css#L140 We may want to just copy the styles for the "create_container_form" element.